crates/esthri/src/rusoto.rs

```diff
@@ -188,11 +232,12 @@ where
     Ok(())
 }
 
-pub async fn create_multipart_upload<T>(
+pub async fn multipart_upload_with_storage_class<T>(
```

Comment:
Looks great, but can we please add the same upload (`_with_storage_class`) function variants for the other public upload functions?